src/sonarcloud.ts

```diff
@@ -25,14 +25,26 @@ export const checkForErrors = (response: any) => {
   }
 };
 
-export const handleErrors = (errors: Error[]) => {
-  for (const error of errors) {
-    if (error.message.includes("No organization for key")) {
+type SonarcloudError = {
+  message?: string;
+  msg?: string;
+};
+
+export const handleErrors = (errors: SonarcloudError[]) => {
+  for (let error of errors) {
+    const errorKeys = Object.keys(error);
+    if (!errorKeys.includes("message") && !errorKeys.includes("msg")) {
```

Comment:
This feels a little more complicated than it might be.  Is there a reason this can't be done with a couple of property checks like this?

```suggestion
    if (!(error.message || error.msg)) {
```